### DISCUSSION TITLE
GPS display with outline (although suppressed in Settings/Appearance)

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#415] [Q&A] [answered]
Just discovered today: 
![2024-01-03 22_09_05-2024-01-03 16 36 35 png - IrfanView (Zoom_ 1878 x 939)](https://github.com/devemux86/cruiser/assets/121806238/2b4a136c-7bfe-4b55-9d51-fca123e4ce25)
Not really important!

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the report.

This will work properly in the next update.

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.38](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
